lib/cli.js

```diff
@@ -207,10 +207,19 @@ Promise.resolve().then(() => {
   return standalone(options)
 }).then((linted) => {
   if (reportNeedlessDisables) {
-    process.stdout.write(needlessDisablesStringFormatter(linted.needlessDisables))
-    if (reportNeedlessDisables === "error") {
+    const hasReportNeedlessDisable = linted.needlessDisables.some((sourceReport) => {
```

Comment:
Based on the type definitions, `needlessDisables` is not *guaranteed* to be on the `linted` object, so you can't call `some` on it without first checking whether it exists. An easy way to fix this with lodash, I think: `_.get(linted, 'needlessDisables', []).some(..)`.